debian/changelog

```diff
@@ -1,4 +1,4 @@
-yq (3.3-0) focal; urgency=medium
+yq (3.3.0) focal; urgency=medium
```

Comment:
> dpkg-source: error: can't build with source format '3.0 (native)': native package version may not have a revision

Reply:
@dukelion did you try '3.0 (quilt)' format with 3.3-0 version pattern?

Reply:
Yes, it didnt work either.